client/src/app/shared/forms/form-validators/user-validators.service.ts

```diff
@@ -23,15 +23,15 @@ export class UserValidatorsService {
     this.USER_USERNAME = {
       VALIDATORS: [
         Validators.required,
-        Validators.minLength(3),
-        Validators.maxLength(20),
-        Validators.pattern(/^[a-z0-9._]+$/)
+        Validators.minLength(1),
+        Validators.maxLength(50),
+        Validators.pattern(/^[a-zA-Z0-9][a-zA-Z0-9-._]*$/)
```

Comment:
Please keep lowercases. With uppercases, it needs more work on backend

Reply:
Ok I'll change it to lowercase, no worries